Guide-‐-Event-Manager.md

```diff
@@ -118,27 +126,32 @@ and with that, you should have everything you need to use events. However, I'll
 ## Queues
 So far we have just been adding events to the base queue. However, there are a few other queues for us to use.
 
-When calling G.E_MANAGER:add_event, you can pass in a second argument for the queue. This is string corresponding to the queue type (default is "base"). The queues are as follows:
+When calling `G.E_MANAGER:add_event`, you can pass in a second argument for the queue. This is a string corresponding to the queue type (default is "base"). The queues are as follows:
 
 - base - The default queue. This is where most events should go.
-- other - Was once used for displaying a video in the demo. Since it's not used, it makes for a prime candidate for putting stuff in, if you want to use blocking but not block the base queue.
+- other - Was once used for displaying a video in the demo. Since it's not used, it makes for a prime candidate for putting stuff in, if you want to use blocking events without clogging up the base queue.
```

Comment:
I would prefer the term `block` then clogging, as it's the proper technical term